src/uibatch.c

```diff
@@ -28,7 +28,8 @@ static uint64_t Ticks;
 
 static void clearline(void)
 {
-    static const size_t linelength = strlen(DistractorFormat) * 2;
+    static size_t linelength;
+    linelength = strlen(DistractorFormat) * 2;
```

Comment:
this is a bit of a cheat, since `DistractorFormat` is a local string literal I think the compiler sees through the `strlen` call and just turns it into a `sizeof` making it a valid static initialization. that's not technically valid C though. the only change I would make here is just drop the `static` modifier and make `linelength` `const` instead